I.write-tests/02-remove-vowels/remove-vowels.test.js

```diff
@@ -1,8 +1,14 @@
-test('remove vowels from word', function () {
+test("remove vowels from word", function() {
+  // Arrange
+  // Given
+  var removeVowels = require("./remove-vowels.js");
```

Comment:
Usually these require lines go at the top of the file, so that it doesn't have to be done in each test, especially since there will typically be multiple tests in each file!

Reply:
Thanks will move it to the top next time